### ISSUE TITLE
When would you plan to release the deeperLab for panoptic segmentation task?

### ISSUE THREAD

**Issue by WenguoLi** [tensorflow/models#6923]
Thank you for sharing the code of [metric for panoptic segmentation ](https://github.com/tensorflow/models/tree/master/research/deeplab/evaluation).
This work is so amazing and I would like to check your code.
When would you plan to release the training code and pretrained models of deeperLab for panoptic segmentation task ?

Thanks

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by kings-landing** [comment]
@aquariusjay Any update on releasing the deeperlab model? Thanks!!

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be an old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
